frb_codegen/src/generator/dart/mod.rs

```diff
@@ -467,53 +620,106 @@ fn generate_api_func(func: &IrFunc, ir_file: &IrFile) -> GeneratedApiFunc {
     }
 }
 
-fn generate_api2wire_func(ty: &IrType, ir_file: &IrFile, block_index: BlockIndex) -> String {
-    if let Some(body) = TypeDartGenerator::new(ty.clone(), ir_file, None).api2wire_body(block_index)
-    {
+fn rust_wire_to_dart_wire(ty: &str) -> Cow<str> {
```

Comment:
I guess we already have some method like `.dart_wire`, so converting rust_wire to dart_wire in a new function may be confusing?